### PR TITLE
Add test to ensure maven central index is downloaded

### PR DESCRIPTION
**Pull request by AObuchow**
Signed-off-by: Andrew Obuchowicz <aobuchow@redhat.com>

### CODE REVIEW

File: org.eclipse.lsp4xml.extensions.maven/pom.xml [hunk 1]
```diff
@@ -80,17 +80,28 @@
...
 		</dependency>
 	</dependencies>
-
+	
 	<build>
 		<plugins>
 			<plugin>
 				<groupId>org.apache.felix</groupId>
 				<artifactId>maven-bundle-plugin</artifactId>
+				<version>6.0.3</version>
```
Comment (AObuchow) [inline org.eclipse.lsp4xml.extensions.maven/pom.xml]: Remove this?

**Comment by mickaelistria** [comment]
Do we still need it now we found a smaller repo with index for testing?

**Comment by AObuchow** [comment]
> Do we still need it now we found a smaller repo with index for testing?

We don't really need it anymore, although I could modify it to reference the smaller repo to detect any regressions with index downloading (or if the repository goes offline). What do you think?

**Comment by mickaelistria** [comment]
>  What do you think?

I don't have an opinion one way or the other. If you think it's valuable enough to work on it now, let's make the change, otherwise, let's discard.

**Comment by AObuchow** [comment]
> > What do you think?
> 
> I don't have an opinion one way or the other. If you think it's valuable enough to work on it now, let's make the change, otherwise, let's discard.

I'll try to quickly make the change, seeing as most of the work is already done.

**Comment by AObuchow** [comment]
> > > What do you think?
> > 
> > 
> > I don't have an opinion one way or the other. If you think it's valuable enough to work on it now, let's make the change, otherwise, let's discard.
> 
> I'll try to quickly make the change, seeing as most of the work is already done.

On second thought, I forgot that we changed some of the API related to the indexer which complicates updating this PR. I'm going to discard this for now.

**Pull request closed without merging**